diagnostic.md

```diff
@@ -6,38 +6,38 @@ Write your answers inside this file, where it's indicated by the comments.
     What command do we use to do that? Use the correct URL for your fork of this
     repository in your answer.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+    git clone https://github.com/tina513/git-basics-diagnostic.git
```

Comment:
Hey Brian,


This question didn't specify which url(HTTP or SSH) we need to put for answer. And the HTTP url I used actually indicate MY repository. So I don't think I missed for this question. Please let me know if I missed any other important info.


Thank you.


________________________________
From: Brian Berzellini <notifications@github.com>
Sent: Tuesday, May 17, 2016 9:42 PM
To: ga-wdi-boston/git-basics-diagnostic
Cc: tina513; Author
Subject: Re: [ga-wdi-boston/git-basics-diagnostic] Git diagnostic from tina (#84)


In diagnostic.md<https://github.com/ga-wdi-boston/git-basics-diagnostic/pull/84#discussion_r63634138>:

> @@ -6,38 +6,38 @@ Write your answers inside this file, where it's indicated by the comments.
>      What command do we use to do that? Use the correct URL for your fork of this
>      repository in your answer.
>
> -    ```sh
> -    <!-- Remove this comment and place your answer here. -->
> +    ```
> +    git clone https://github.com/tina513/git-basics-diagnostic.git


git clone git@github.com/tina513/git-basics-diagnostic.git. the SSH url for YOUR forked repository.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ga-wdi-boston/git-basics-diagnostic/pull/84/files/b39a80a65bb583bc13c68556d849d65f13a1b476#r63634138>